### PR TITLE
KFSTP-2121 fix deleting items from saved Travel docs

### PR DESCRIPTION
**Pull request by kkronenb**

### CODE REVIEW

File: kfs-tem/src/main/java/org/kuali/kfs/module/tem/document/TravelDocumentBase.java [hunk 1]
```diff
@@ -1336,8 +1336,9 @@ public List buildListOfDeletionAwareLists() {
         List managedLists = super.buildListOfDeletionAwareLists();
 
         managedLists.add(getImportedExpenses());
-        managedLists.add(getActualExpenses());
```
Comment (smith750) [inline kfs-tem/src/main/java/org/kuali/kfs/module/tem/document/TravelDocumentBase.java]: What was the deal with Actual Expense?

**Comment by kfsci** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.kfs.kuali.org/job/Unit-Test-kfs-oracle/50/
Test PASSed.

**Comment by kfsci** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.kfs.kuali.org/job/Unit-Test-kfs-mysql/68/
Test PASSed.

File: kfs-tem/src/main/java/org/kuali/kfs/module/tem/document/TravelDocumentBase.java (hunk 1 above)
Comment (kkronenb) [inline kfs-tem/src/main/java/org/kuali/kfs/module/tem/document/TravelDocumentBase.java]: the detail lines would not delete either, only the parent was deletable.

**Pull request merged**